llvm/tools/llvm-exegesis/lib/Target.cpp

```diff
@@ -39,12 +39,18 @@ const char *
 ExegesisTarget::getIgnoredOpcodeReasonOrNull(const LLVMState &State,
                                              unsigned Opcode) const {
   const MCInstrDesc &InstrDesc = State.getIC().getInstr(Opcode).Description;
-  if (InstrDesc.isPseudo() || InstrDesc.usesCustomInsertionHook())
-    return "Unsupported opcode: isPseudo/usesCustomInserter";
-  if (InstrDesc.isBranch() || InstrDesc.isIndirectBranch())
-    return "Unsupported opcode: isBranch/isIndirectBranch";
-  if (InstrDesc.isCall() || InstrDesc.isReturn())
-    return "Unsupported opcode: isCall/isReturn";
+  if (InstrDesc.isPseudo())
```

Comment:
All the changes in this file seem to be formatting changes that are now irrelevant to the PR? They should probably be reverted.

Reply:
I would prefer these changes go in a separate patch.